hack/ginkgo-e2e.sh

```diff
@@ -130,13 +128,12 @@ else
     "--auth_config=${AUTH_CONFIG:-}"
     "--cert_dir=${CERT_DIR:-}"
   )
-  host="https://${KUBE_MASTER_IP-}"
 fi
 
 # Use the kubectl binary from the same directory as the e2e binary.
 export PATH=$(dirname "${e2e}"):"${PATH}"
 "${e2e}" "${auth_config[@]:+${auth_config[@]}}" \
-  --host="$host" \
+  --host="https://${KUBE_MASTER_IP-}" \
```

Comment:
In cluster/libvirt-coreos/util.sh you use http://${KUBE_MASTER_IP-}:8080 which is now inconsistent with what's used here (and was used here previously) - was this change intentional?